mlflow/utils/process.py

```diff
@@ -18,7 +18,7 @@ def exec_cmd(
     throw_on_error -- if true, raises an Exception if the exit code of the program is nonzero
     env -- additional environment variables to be defined when running the child process
     cwd -- working directory for child process
-    stream_output -- if true, does not capture standard output and error; if false, captures these
+    capture_output -- if False, does not capture standard output and error; if True, captures these
```

Comment:
```suggestion
    capture_output -- if True, captures standard output and error; if False, does not captures these
```

nit: can we swap `True` and `False`?